start-tests.sh

```diff
@@ -355,10 +355,23 @@ if [[ "$SKIP_UI_TEST" == false ]]; then
   fi
 
   log_color "cyan" "Running console cypress tests."
-  # NOTE: This reduces intermittent failures due to the console not being ready.
-  #       In the future we should replace the sleep with a check for the console being ready.
-  echo "Waiting 60 seconds for console to be ready..."
-  sleep 60; 
+
+  echo "Waiting up to 10 minutes for console pods to reach Running status"
+  RUNNING="false"
+  ATTEMPTS=0
+  MAX_ATTEMPTS=20
+  INTERVAL=30
+  while [[ "${RUNNING}" == "false" ]] && (( ATTEMPTS != MAX_ATTEMPTS )); do
+    RUNNING_PODS_COUNT=$(`oc get pods -n open-cluster-management -l app=console-chart-v2 --field-selector=status.phase==Running --no-headers | wc -l`)
+    if [ $RUNNING_PODS_COUNT == 2 ]; then
```

Comment:
The test should pass with only 1 instance of the console running. Waiting for 2 might be safer but could cause unnecessary wait.
**It works either way, so just thinking out loud.**

Reply:
Probably best to just see if at least 1 is running. can change to >= 1.